controllers/packetmachine_controller.go

```diff
@@ -476,7 +482,8 @@ func (r *PacketMachineReconciler) reconcileDelete(ctx context.Context, machineSc
 		return ctrl.Result{}, fmt.Errorf("%w: %s", ErrMissingDevice, packetmachine.Name)
 	}
 
-	if _, err := r.PacketClient.Devices.Delete(device.ID, force); err != nil {
+	apiRequest := r.PacketClient.DevicesApi.DeleteDevice(nil, *device.Id).ForceDelete(force)
```

Comment:
ctx here too (and everywhere where we can now honor it!) - that's the last time I'll call it out 😄